src/groups/mqb/mqbs/mqbs_filebackedstorage.h

```diff
@@ -203,7 +203,7 @@ class FileBackedStorage BSLS_KEYWORD_FINAL : public ReplicatedStorage {
     bmqp::SchemaLearner::Context d_schemaLearnerContext;
     // Context for replicated data.
 
-    const bool d_hasReceipts;
+    bool d_hasReceipts;
```

Comment:
We set this flag once and didn't update it before. It is also used here for early return:

https://github.com/bloomberg/blazingmq/blob/c0f8832ce69bebefd5eaaf3637fe95fa8bb9f138/src/groups/mqb/mqbs/mqbs_filebackedstorage.cpp#L216-L221